facade/nodeFacade.go

```diff
@@ -348,6 +355,38 @@ func (nf *nodeFacade) GetTransactionsPoolNonceGapsForSender(sender string) (*com
 	return nf.apiResolver.GetTransactionsPoolNonceGapsForSender(sender, accountResponse.Nonce)
 }
 
+// GetSelectedTransactions will simulate a SelectTransactions, and it will return the corresponding hash of each selected transaction
+func (nf *nodeFacade) GetSelectedTransactions() (*common.TransactionsSelectionSimulationResult, error) {
+	currentRootHash := nf.blockchain.GetCurrentBlockRootHash()
+	if currentRootHash == nil {
+		return nil, ErrNilCurrentRootHash
+	}
+
+	blockHeader := nf.blockchain.GetCurrentBlockHeader()
+	if blockHeader == nil {
+		return nil, ErrNilBlockHeader
+
+	}
+
+	epoch := blockHeader.GetEpoch()
+	rootHashHolder := holders.NewRootHashHolder(currentRootHash, core.OptionalUint32{Value: epoch, HasValue: true})
+
+	// NOTE: keep in mind that the selection simulation can be affected by other API requests which might alter the trie
```

Comment:
```suggestion
	// TODO: keep in mind that the selection simulation can be affected by other API requests which might alter the trie
```
suggestion to add it as `TODO` since we usually filter by "TODO" string when search for todos or notes